src/utils.py

```diff
@@ -0,0 +1,55 @@
+from nes_py.wrappers import JoypadSpace
+import gym_super_mario_bros
+from gym_super_mario_bros.actions import SIMPLE_MOVEMENT
+import numpy as np
+import matplotlib.pyplot as plt
+from IPython import display as ipythondisplay
+from tqdm import tqdm
+from pyvirtualdisplay import Display
+
+class Utils:
```

Comment:
We can rename this class since it's leaning towards just being the environment - perhaps 
"SuperMarioBrosDataset"